src/pages/TeamSettings/CloudHooks.vue

```diff
@@ -323,7 +323,7 @@ export default {
       <div class="py-1 mr-2 flex">
         <v-text-field
           v-model="search"
-          class="rounded-none elevation-1"
+          class="rounded-0 elevation-1"
```

Comment:
what's the difference between `rounded-none` and `rounded-0`?

Reply:
technically nothing, `rounded-0` is built into Vuetify, so since we're already importing the package we may as well use their utility classes instead of rewriting our own!